pkgs/development/interpreters/nextflow/default.nix

```diff
@@ -46,9 +45,14 @@ stdenv.mkDerivation rec {
   postFixup = ''
     wrapProgram $out/bin/nextflow \
       --prefix PATH : ${lib.makeBinPath nativeBuildInputs} \
-      --set JAVA_HOME ${openjdk17.home}
+      --set JAVA_HOME ${openjdk.home}
   '';
 
+  passthru.tests.version = testers.testVersion {
+    package = nextflow;
+    command = "HOME=$TMPDIR nextflow -version";
+  };
```

Comment:
Think we could get away with a `nextflow run hello` too?

Reply:
Mh, I think this needs internet access, right? I tried to run this and this yields:

```bash
❯ nix build path:.#nextflow.tests.hello
error: builder for '/nix/store/k7ic1jfrcmyiaxckk5ziqgcjvic1ckaw-nextflow-run-hello.drv' failed with exit code 1;
       last 10 log lines:
       > sourcing setup hook '/nix/store/cickvswrvann041nqxb0rxilc46svw1n-prune-libtool-files.sh'
       > sourcing setup hook '/nix/store/xyff06pkhki3qy1ls77w10s0v79c9il0-reproducible-builds.sh'
       > sourcing setup hook '/nix/store/aazf105snicrlvyzzbdj85sx4179rpfp-set-source-date-epoch-to-latest.sh'
       > sourcing setup hook '/nix/store/gps9qrh99j7g02840wv5x78ykmz30byp-strip.sh'
       >
       >  N E X T F L O W   ~  version 24.04.4
       >
       > Pulling nextflow-io/hello ...
       > WARN: Cannot read project manifest -- Cause: api.github.com
       > Cannot find `nextflow-io/hello` -- Make sure exists a GitHub repository at this address `https://github.com/nextflow-io/hello`
       For full logs, run 'nix log /nix/store/k7ic1jfrcmyiaxckk5ziqgcjvic1ckaw-nextflow-run-hello.drv'.
```

```diff
-  passthru.tests.version = testers.testVersion {
-    package = nextflow;
-    command = "HOME=$TMPDIR nextflow -version";
+  passthru.tests = {
+    version = testers.testVersion {
+      package = nextflow;
+      command = "HOME=$TMPDIR nextflow -version";
+    };
+    hello = runCommand "nextflow-run-hello" {nativeBuildInputs = [nextflow];} ''
+      HOME=$TMPDIR
+      nextflow run hello
+      touch $out
+    '';
   };
```

Reply:
I have added a `hello` test by simply downloading the necessary pipeline upfront.